### ISSUE TITLE
(recover) unconscious bug

### ISSUE THREAD

**Issue by S-fly** [EpochModTeam/DayZ-Epoch#1522]
There is a bug related to the loss of consciousness. Often (always?) if you faint and change your position afterwards, you will spawn at the position where you fainted after a relog. Also if you don’t relog, the AI of missions will try to attack you at the position where you fainted and not at your new position. You can solve this by entering and leaving a vehicle. Can someone confirm?

**Comment by seb3sec** [comment]
I did some testing to get rid of this bug and figured out that the >> player playActionNow "Die"; << seems to cause this bug. After this command has been executed the server only updates the position with setPos... once. As nkukard mentioned entering a vehicle also fixes this. 

I'll see which move/action can be used to replace this.

**Comment by SplenectomY** [comment]
It seems very likely that the issue starts in this file somewhere /compile/server_playerSync.sqf. Check your log for any of the debug errors referenced in this file. A likely cause is the player playActionNow "Die"; animation causing the player to be a null object, but try to comment out the animation all together and see if the issue persists.

**Comment by seb3sec** [comment]
No, the playerSync works fine. The bug we're talking about only stops updating the position, gear content and everything else is tracked correctly. There are no errors regarding this in the RPT.

I've written a little test suite for this and one part of it is that you can use a smoke action which requests a smoke grenade drop on your position known by the server. So this basically puts a smoke shell where you're standing. If you get knocked out and move away and use this action again, the smoke will be placed where you've been unconscious. I've also added a animationState display which shows what animation is the server seeing on you and after knockout it's permanently AdthPercMstpSrasWrflDnon_1 (Death, Standing, Step speed, raised Weapon, Rifle in Hand).

**Comment by seb3sec** [comment]
I've tried a lot of dying, lying and sleeping animations now and they all seem to get stuck at one point. After 2 hours of coding a workaround I just discovered that the DayZ Mod uses a very similiar variant which uses the prone animation. 

Anyone knows if its allowed to port over the code from 1.8.4 to Epoch?
(Not sure how to understand that "licensed to" here: https://github.com/DayZMod/DayZ/blob/Development/SQF/dayz_code/system/scheduler/sched_medical.sqf#L1 - is it allowed to modify under DML or is this licensed?)
